### PR TITLE
Add KIN blockchain support

### PR DESCRIPTION
**Pull request by vikmeup**
🚀

### CODE REVIEW

File: src/interface/TWCoinTypeConfiguration.cpp [hunk 1]
```diff
@@ -196,6 +200,7 @@ TWString *_Nonnull TWCoinTypeConfigurationGetID(enum TWCoinType type) {
     case TWCoinTypeStellar: string = "stellar"; break;
     case TWCoinTypeAion: string = "aion"; break;
     case TWCoinTypeNEO: string = "neo"; break;
+    case TWCoinTypeKIN: string = "KIN"; break;
```
Comment (madcake) [inline src/interface/TWCoinTypeConfiguration.cpp]: Why upper case?

File: src/Coin.cpp [hunk 2]
```diff
@@ -127,6 +128,7 @@ bool TW::validateAddress(TWCoinType coin, const std::string& string) {
         return Ripple::Address::isValid(string);
 
     case TWCoinTypeStellar:
+    case TWCoinTypeKIN:
         return Stellar::Address::isValid(string);
```
Comment (madcake) [inline src/Coin.cpp]: Is it correct in long term?

Comment (vikmeup) [reply src/Coin.cpp]: Yes, Kin is a fork of stellar

File: JNI/cpp/StellarPassphrase.h [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+//
+
+#ifndef JNI_TW_STELLARPASSPHRASE_H
+#define JNI_TW_STELLARPASSPHRASE_H
+
+#include <jni.h>
+#include <TrustWalletCore/TWBase.h>
+
+TW_EXTERN_C_BEGIN
+
```
Comment (madcake) [inline JNI/cpp/StellarPassphrase.h]: Why empty?

File: include/TrustWalletCore/TWStellarPassphrase.h [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include "TWBase.h"
+
+TW_EXTERN_C_BEGIN
+
+TW_EXPORT_ENUM()
+enum TWStellarPassphrase {
+    TWStellarPassphraseStellar     /* "Public Global Stellar Network ; September 2015" */,
```
Comment (madcake) [inline include/TrustWalletCore/TWStellarPassphrase.h]: Remove or add spaces|tabs.

File: src/Stellar/Signer.cpp [hunk 5]
```diff
@@ -21,7 +21,7 @@ std::string Signer::sign() const noexcept {
     auto encoded = encode(input);
 
     auto encodedWithHeaders = Data();
-    auto publicNetwork = std::string("Public Global Stellar Network ; September 2015");  // Header
+    auto publicNetwork = std::string(input.passphrase());  // Header
```
Comment (madcake) [inline src/Stellar/Signer.cpp]: Can not necessarily wrap in a string?

File: JNI/cpp/StellarPassphrase.h (hunk 3 above)
Comment (vikmeup) [reply JNI/cpp/StellarPassphrase.h]: I think this is just a header file? not sure

Comment (alejandro-isaza) [reply JNI/cpp/StellarPassphrase.h]: It's an enum with no exported methods.

**Pull request merged**